build/tasks.js

```diff
@@ -1,43 +1,40 @@
 // Any copyright is dedicated to the Public Domain.
 // http://creativecommons.org/publicdomain/zero/1.0/
 
-/* eslint global-require: 0 */
+import chokidar from 'chokidar';
 
 /**
  * Use need to allow lazy loading of modules for all tasks. Generally,
  * we assume `build-config.js` exists, however *during* the build process
  * this might not be true, which will break certain native module imports that
  * try to eagerly load this file.
  */
+ /* eslint global-require: 0 */
+
+const Lazy = {
+  buildDeps: () => require('./task-build-deps').default(),
+  config: options => require('./task-config-builder').default(options),
+  build: args => require('./task-build').default(args),
+  run: args => require('./task-run').default(args),
+  test: args => require('./task-test').default(args),
+  package: () => require('./task-package').default(),
+  clean: () => require('./task-clean-package').default(),
+};
```

Comment:
It looked pretty complicated to properly maintain the arguments/options and their order passed into require(foo).default(...) calls. Making sure there's only one way to invoke those default imports should prevent bugs where we don't pass in the correct args.